src/views/Home.jsx

```diff
@@ -1,16 +1,23 @@
 import './Home.css';
-
+import { SingleList } from '../components';
 export function Home({ data, setListPath }) {
 	return (
 		<div className="Home">
 			<p>
 				Hello from the home (<code>/</code>) page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `lists` array
-				 * so we can see which lists the user has access to.
-				 */}
+				{/* insert path as prop later */}
+				{data &&
+					data.map((list) => {
+						return (
+							<SingleList
+								key={list.id}
+								name={list.name}
+								setListPath={setListPath}
```

Comment:
Now that we have the right data structure in Firebase, go ahead and add the path here! This should work with that change! 👍

Reply:
Added it! Also, there was an error that the keys weren't unique so used the randomUUID() method to pass into the keys for now.